test/server/cards/02_SHD/units/FinalizerMightOfTheFirstOrder.spec.ts

```diff
@@ -40,16 +40,16 @@ describe('Finalizer, Might of the First Order', function() {
                 context.player1.clickPrompt('Done');
 
                 expect(context.player1).toHavePrompt('Choose a unit to capture with Iden Versio');
-                expect(context.player1).toBeAbleToSelectExactly([context.steadfastBattalion, context.battlefieldMarine, context.isbAgent, context.superlaserTechnician]);
-                context.player1.clickCard(context.superlaserTechnician);
+                expect(context.player1).toBeAbleToSelectExactly([context.steadfastBattalion, context.battlefieldMarine, context.zuckuss, context._4lom]);
```

Comment:
Just in case we accidentally break this behavior later due to engine changes and cause the Zuckuss / 4-LOM part to fail, do you mind adding a comment above this explaining why they're included and what the expected behavior is (i.e., captured not defeated)